test/wdio/svg-class/cmp.test.tsx

```diff
@@ -1,15 +1,28 @@
-import { setupDomTests, waitForChanges } from '../util';
+import { Fragment, h } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
 
 describe('svg class', () => {
-  const { setupDom, tearDownDom } = setupDomTests(document);
-  let app: HTMLElement;
-
-  beforeEach(async () => {
-    app = await setupDom('/svg-class/index.html');
+  beforeEach(() => {
+    render({
+      template: () => (
+        <>
+          <style>
+            {`.primary circle {
+                fill: red;
+              }
+              .primary rect {
+                fill: blue;
+              }`}
+          </style>
+          <svg-class></svg-class>
+        </>
+      ),
+    });
   });
-  afterEach(tearDownDom);
 
   it('toggles svg class', async () => {
+    const app = document.body;
+    await $('svg-class').waitForStable();
```

Comment:
FWIW, I had some flakiness trying to run this test locally - changing this to
```suggestion
    await $('svg-class').waitForExist();
```
seemed to help.

@christian-bromann is there much a difference between these two in practice? Do they 'wait' for the same amount of time?

Reply:
"stable" in this context means whether or not the element moves or not. I think we can just check for `waitForExist` as the location of the element mostly doesn't matter since they aren't animated.

Reply:
ah ok, I sort of imputed my own definition of `waitForStable` which had to do more with no more children being added or something. Good to know!